.env

```diff
@@ -0,0 +1,2 @@
+publicKey=pk_test_fdb8f5d097ae34d157c37dade5a73e773a2e59c2
```

Comment:
You should totally avoid commit application settings and secret keys to source control

Reply:
Thanks. I forgot to gitignore it.